src/views/userProgressView.js

```diff
@@ -60,3 +59,8 @@ export const createScoreElement = (numberOfCorrects) => {
     `;
   return element;
 };
+export const createTimeElement = () => {
+  const element = document.createElement('div');
+  element.classList.add('progress-time-item');
```

Comment:
Please use an id for `progress-time-item` and store it in constants, like the rest of the elements do